`deps.ts` breaks packaging with pkg

The FAQ says to use https://github.com/zeit/pkg for building single binaries but when attempting to run commands on the packaged binary I get an error that looks like

    Error: Cannot find module './parse'
    1) If you want to compile the package/file into executable, please pay attention to compilation warnings and specify a literal in 'require' call. 2) If you don't want to compile the package/file into executable and want to 'require' it from filesystem (likely plugin), specify an absolute path in 'require' call using process.cwd() or process.execPath.
        at Function.Module._resolveFilename (pkg/prelude/bootstrap.js:1269:46)
        at Module.require (pkg/prelude/bootstrap.js:1153:31)
        at fetch (/snapshot/ssmenv/node_modules/@oclif/parser/lib/deps.js:34:16)
        at Object.get parse [as parse] (/snapshot/ssmenv/node_modules/@oclif/parser/lib/deps.js:12:12)
        at Object.parse (/snapshot/ssmenv/node_modules/@oclif/parser/lib/index.js:20:36)
        at Main.parse (/snapshot/ssmenv/node_modules/@oclif/command/lib/command.js:64:41)

The distributed `deps.js` from `@oclif/parser` is the only place I can find this pattern. After changing the export in the distributed `deps.js` to look like:

    exports.deps = {
      // local
      args: require('./args'),
      flags: require('./flags'),
      parse: require('./parse'),
      validate: require('./validate'),
      renderList: require('./list').renderList,
      errors: require('./errors'),
      chalk: require('chalk'),
    };

I can no longer reproduce the in my testing.

--- a/src/deps.ts
+++ b/src/deps.ts
@@ -1,4 +1,3 @@
-// tslint:disable no-implicit-dependencies
 import chalk = require('chalk')
 
 import args = require('./args')
@@ -10,27 +9,18 @@ import validate = require('./validate')
 
 export const deps = {
   // local
-  get args(): typeof args { return fetch('./args') },
-  get flags(): typeof flags { return fetch('./flags') },
-  get parse(): typeof parse { return fetch('./parse') },
-  get validate(): typeof validate { return fetch('./validate') },
-  get renderList(): typeof list.renderList { return fetch('./list').renderList },
-  get errors(): typeof errors { return fetch('./errors') },
+  get args(): typeof args { return args },
+  get flags(): typeof flags { return flags },
+  get parse(): typeof parse { return parse },
+  get validate(): typeof validate { return validate },
+  get renderList(): typeof list.renderList { return list.renderList },
+  get errors(): typeof errors { return errors },
 
   // remote
   get chalk(): typeof chalk.default | undefined {
     try {
-      return fetch('chalk').default
+      return chalk.default
       // tslint:disable-next-line no-unused
     } catch (err) {}
   },
-}
-
-const cache: any = {}
-
-function fetch(s: string) {
-  if (!cache[s]) {
-    cache[s] = require(s)
-  }
-  return cache[s]
 }

